### DISCUSSION TITLE
[Improvement]: Add docker image and commit based preview deployments

### DISCUSSION THREAD

**Discussion by peaklabs-dev** [coollabsio/coolify#3159] [Improvement Requests]
### Description

Currently, we only get preview deployments for pull requests. I would like to see preview deployments for docker images and commits as well.

### Minimal Reproduction (if possible, example repository)

- [ ] Add commit-based preview deployments -> Whenever there is a new commit on the selected branch, coolify will automatically create a preview deployment with the selected commit.
- [ ] Add docker image based preview deployments -> Whenever there is a new image of the docker container on the server, a preview deployment is automatically created. Goes hand in hand with image updates, see: https://github.com/coollabsio/coolify/issues/2500

### Version

-

**Comment by KaelWD** [comment]
I'd be happy with just an api endpoint to trigger a preview deployment from a specific image version.
